.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Pi-hole: A black hole for Internet advertisements
+*  (c) 2022 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license.
+*/
+
+declare(strict_types=1);
+
+$finder = PhpCsFixer\Finder::create()
+    ->ignoreDotFiles(false)
+    ->ignoreVCSIgnored(true)
+    ->exclude('scripts/vendor')
+    ->in(__DIR__)
+;
+
+$config = new PhpCsFixer\Config();
+$config
+    ->setRules(array(
+        '@Symfony' => true,
+        'array_syntax' => array('syntax' => 'long'),
+    ))
+    ->setFinder($finder)
```

Comment:
We should add `->setLineEnding(PHP_EOL)` here.

With this config, PhpCsFixer will use the PHP line ending locally. This will ensure `LF` on Linux systems and `CRLF` on Windows systems and avoid warning messages like: 
```
warning: LF will be replaced by CRLF in api.php.
The file will have its original line endings in your working directory
```